doc/TBG_macros.cfg

```diff
@@ -188,6 +188,19 @@ TBG_restart="--restart"
 # Note: does currently not work with `Radiation` plugin
 TBG_softRestarts="--softRestarts 5000"
 
+# ISAAC server and rendering interval settings
+TBG_isaac="--isaac.period 1 --isaac.name !TBG_jobName --isaac.url <server_url>"
+
+# ISAAC JPEG quality (1..100). Smaller values are faster sent, but of lower quality
+TBG_isaac_quality="--isaac.quality 80"
```

Comment:
But its "downstream" from the internet bottleneck point of view which in reality never caused trouble for me since >1 (10) (100) MBit/s was so far always available.